Задание 1.27. Как делишки?

MBLVRD opened the pull request:
Ну давай попробуем вот так. Похоже, что мы не замечали слона в комнате и нужно было нажать Compare & Pull request и перестать бояться атомной бомбы.

MBLVRD committed:
Задание 1.27. Как делишки?

MBLVRD committed:
Увлёкся, 1.30 сделал вместе с 1.31 + модифицировал

MBLVRD committed:
Это просто для себя, надеюсь не запрещено

MBLVRD commented:
Да блин, мне не нравится, что он все задания в кучу пихает(

MBLVRD committed:
Так, это по заданию 1.32 Надеюсь, что видно норм

MBLVRD committed:
1.32

obopodo commented on Work/pcost.py:
```diff
@@ -1,3 +1,31 @@
+        with open(filename) as f:
+            rows = csv.reader(f)
+            headers = next(rows)
+            for row in rows:
```
у тебя до этого было хорошо сделано, когда ты три пременных из одной строки инитил, тут тоже можно так:

```suggestion
            for name, shares, price in rows:
```

obopodo replied:
ну и дальше соответственно поправить

MBLVRD replied:
@obopodo хорошо, спасибо. Сфокусировался на задаче, а там было просто переделать из одного в другое.

obopodo reviewed (commented):
В 1.31-32 стоит разбить общее полотно на несколько функций. Две тут точно можно выделить: первая как есть - проверяет, что файл есть и выдаёт ответ, а внутри себя вызвыает другую, которая уже открывает файл и делает вычисления по нему

MBLVRD committed:
Update Work/pcost.py

В общем, мне сложно тогда понять, как перехватывать ошибку и продолжать цикл, пропуская ошибочную линию, потмоу что ошибка происходит на уровне разбивки по переменных, тогда как если писать в row, то он создастся без проблемы, и ошибка возникнет там, где я её жду. Если сможешь помочь - буду рад.

Co-authored-by: obopodo <42578589+obopodo@users.noreply.github.com>
